samples/mpu/mpu_stack_guard_test/sample.yaml

```diff
@@ -5,7 +5,8 @@ tests:
     tags: apps
     harness: console
     # We do not handle segfaults here
-    platform_exclude: native_posix
+    arch_exclude: posix riscv32 nios2
+    platform_exclude: qemu_cortex_m3
```

Comment:
this patch needs to be a REVERTME with a bug filed on it, this could be a symptom of something serious (or not)